presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -5959,6 +5959,38 @@ public void testTry()
     }
 
     @Test
+    public void testLambdaFunctionWithTry()
```

Comment:
Merge it into `testTry`. Add comment like `lambda function with try`.

I would also like to see some query use try and lambda on TPCH dataset.